libs/designer/src/lib/ui/panel/recommendation/recommendationPanelContext.tsx

```diff
@@ -181,6 +185,7 @@ export const RecommendationPanelContext = (props: RecommendationPanelContextProp
               onOperationClick={onOperationClick}
               isLoading={isLoadingOperations || isLoadingOperationGroup}
               displayRuntimeInfo={displayRuntimeInfo}
+              respectActionsFilter={!hideActionTypeFilter}
```

Comment:
Nit but I'd prefer the prop here to be inverted, ie:
`ignoreActionsFilter={hideActionTypeFilter}`

Reply:
Makes sense, done.